packages/destination-actions/src/destinations/launchdarkly-audiences/index.ts

```diff
@@ -0,0 +1,68 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import syncAudience from './syncAudience'
+import { defaultValues } from '@segment/actions-core'
+import { CONSTANTS } from './constants'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Launchdarkly Audiences',
+  slug: 'actions-launchdarkly-audiences',
+  mode: 'cloud',
+  description: 'Sync Segments audiences to LaunchDarkly Big Segments.',
+
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      apiKey: {
+        label: 'API Key provided by the LaunchDarkly integration',
+        description:
+          'API Key used for [LaunchDarkly API authorization](https://app.launchdarkly.com/settings/authorization).',
```

Comment:
yup - `createSegment`, `updateIncluded` for sure but I don't think `updateName` or `updateTags` since it doesn't look like we actually support updating those fields

for segment in particular we use the same input for both `cohortId` and `cohortName` so it's maybe not as relevant here, but for the other integrations the "upsert" function is kind of a misnomer and we are really only updating the users/contexts

Reply:
Thanks for checking. I just updated my suggestion to reflect this.